site/docs/guidelines/checkboxs-radiobuttons-toggles.mdx

```diff
@@ -0,0 +1,131 @@
+    );
+  }}
+</Playground>
+
+## When to use checkboxes?
+<Link href="../components/checkbox" >HDS checkbox component</Link>
+<br /><br />
+Checkboxes allow the user to select zero, one, or multiple selections from a list of options. In other words, each checkbox is independent of the other checkboxes, i.e. checking one does not uncheck the others.
+
+A stand-alone checkbox is used for a single option that the user can turn on or off.
+
+<b>Example use: </b>Single selection confirming on/off selection
+
+<Playground>
+  {() => {
+    const [checked, setChecked] = React.useState(false);
+    return <Checkbox id="checkbox" label="I have a loyalty card" checked={checked} onChange={() => setChecked(!checked)} />;
+  }}
+</Playground>
+
+<br />
+<b>HDS recommends using checkboxes when the user is required to make more than two choices or confirm a single choice. </b>
+<br /><br />
+<b>Example use:</b> Selecting interests < br />
+
+<Playground>
+  {() => {
+    const [checkedItems, setCheckedItems] = React.useState({ "v-checkbox1": true, "v-checkbox2": true });
+    const onChange = (event) => {
+      setCheckedItems({ ...checkedItems, [event.target.name]: event.target.checked });
+    };
+    return (
+      <SelectionGroup label="Select your interests">
+        <Checkbox id="v-checkbox1" name="v-checkbox1" label="Music" checked={checkedItems["v-checkbox1"]} onChange={onChange} />
+        <Checkbox id="v-checkbox2" name="v-checkbox2" label="Arts and museums" checked={checkedItems["v-checkbox2"]} onChange={onChange} />
+        <Checkbox id="v-checkbox3" name="v-checkbox3" label="Sports and outdoor activities" checked={checkedItems["v-checkbox3"]} onChange={onChange} />
+        <Checkbox id="v-checkbox4" name="v-checkbox4" label="Cooking" checked={checkedItems["v-checkbox4"]} onChange={onChange} />
+      </SelectionGroup>
+    );
+  }}
+</Playground>
+
+<br />
+<b>HDS recommends using checkboxes when the user can select multiple options that are not excluding other options. </b>
+
+## When to use toggles or switches?
+<Link href="../components/toggle">HDS toggle component</Link>
+<br /><br />
+Toggle switches are digital on/off switches. They prompt users to choose between two mutually exclusive options and always have a default value. Toggle buttons have been inspired by real-world physical switches (e.g. a light switch). Therefore users expect something to happen immediately when the toggle button is activated. Toggle buttons should do three things — change states, show the current state, and reveal unselected options. If your toggle button doesn’t do all these things, it has poor usability. Toggles are best used for switching between two opposing states of system functionality and preferences. They are similar to radio buttons or checkboxes in that they allow users to select from only two options.
+<br /><br />
+<b>Example use:</b> Selecting between light and dark mode, including past events to event listing.
+
+<Playground>
```

Comment:
Maybe we could change the theme of the example or something when the user uses the toggle button. Then it would be in line with the guideline text.